x/qgb/abci_test.go

```diff
@@ -526,3 +520,68 @@ func TestDataCommitmentCreationCatchup(t *testing.T) {
 		assert.Equal(t, want[i].Nonce, dc.Nonce)
 	}
 }
+
+// TestPruning tests the pruning mechanism by:
+// 1. Generating a set of attestations
+// 2. Running the QGB EndBlocker
+// 3. Verifying that the expired attestations are pruned
+func TestPruning(t *testing.T) {
+	input, ctx := testutil.SetupFiveValChain(t)
+	qgbKeeper := *input.QgbKeeper
+	// set the data commitment window
+	window := uint64(101)
+	qgbKeeper.SetParams(ctx, types.Params{DataCommitmentWindow: window})
+	initialBlockTime := ctx.BlockTime()
+	// make the interval between blocks being 10 minutes
```

Comment:
```suggestion
	// make the interval between blocks 10 minutes
```

or alternatively remove the comment and document via code:

```go
blockInterval := 10 * time.Minute
ctx = testutil.ExecuteQGBHeightsWithTime(ctx, qgbKeeper, 1, 1626, blockInterval)
```

Reply:
https://github.com/celestiaorg/celestia-app/pull/1823